plugins/org.eclipse.emfcloud.modelserver.glsp.integration/src/org/eclipse/emfcloud/modelserver/glsp/operations/handlers/EMSChangeBoundsOperationHandler.java

```diff
@@ -13,30 +13,34 @@
 import java.util.HashMap;
 import java.util.Map;
 
-import org.eclipse.emfcloud.modelserver.glsp.notation.Shape;
 import org.eclipse.emfcloud.modelserver.glsp.notation.integration.EMSNotationModelServerAccess;
 import org.eclipse.emfcloud.modelserver.glsp.notation.integration.EMSNotationModelState;
+import org.eclipse.glsp.server.emf.model.notation.Shape;
 import org.eclipse.glsp.server.operations.ChangeBoundsOperation;
 import org.eclipse.glsp.server.types.ElementAndBounds;
 import org.eclipse.glsp.server.types.GLSPServerException;
 
-public class EMSChangeBoundsOperationHandler
-   extends EMSBasicOperationHandler<ChangeBoundsOperation, EMSNotationModelServerAccess> {
+import com.google.inject.Inject;
+
+public class EMSChangeBoundsOperationHandler extends AbstractEMSOperationHandler<ChangeBoundsOperation> {
+
+   @Inject
```

Comment:
This operation handler injects the `*Notation*` versions of the modelState and the modelServerAccess, and to avoid unsafe casting, we inject here.
We could also provide `notation` specific `*OperationHandler`s though.